testsuite/features/step_definitions/common_steps.rb

```diff
@@ -546,6 +546,14 @@
   raise unless output[:stdout].include? '[I] SLE-Module-Server-Applications15-Pool for x86_64 Server Applications Module 15 x86_64 [sle-module-server-applications15-pool-x86_64]'
 end
 
+Then(/^the SLE15SP2 base products should be added$/) do
```

Comment:
I'm not very happy with this way of checking added channels, but... not for this PR.
I would instead have a unique step definition, with a product name variable, then a map, where the product is the key and the list of channels are the value, and we just iterate on them.

Reply:
I'll let you implement that change later then :wink:

Reply:
https://github.com/SUSE/spacewalk/issues/12837